provider/pact-jvm-provider-maven/README.md

```diff
@@ -394,6 +395,31 @@ For example:
 </plugin>
 ```
 
+Here is how you configure the plugin to use basic authentication for verifying pacts
```

Comment:
This should be: Here is how you configure the plugin to use bearer token authentication for verifying pacts

Reply:
👍 thanks!